README.md

```diff
@@ -50,54 +50,31 @@ Follow these steps to create the required services and run the notebook locally.
 
 1. [Setup HDP platform](#1-set-up-hdp-platform)
 2. [Setup HDP Search](#2-set-up-hdp-search)
-3. [Setup DSX Local](#3-set-up-dsx-local)
-4. [Setup a Livy gateway](#4-set-up-a-livy-gateway)
-5. [Download the Solr Spark connector](#5-download-the-solr-spark-connector)
-6. [Clone the repo](#6-clone-the-repo)
-7. [Download and move the data to HDFS](#7-download-and-move-the-data-to-hdfs)
-8. [Setup python plugins](#8-setup-python-plugins)
-9. [Launch the notebook](#9-launch-the-notebook)
-10. [Run the notebook](#10-run-the-notebook)
+3. [Setup DSX Desktop](#3-set-up-dsx-desktop)
+4. [Download the Solr Spark connector](#5-download-the-solr-spark-connector)
+5. [Clone the repo](#6-clone-the-repo)
+6. [Download and move the data to HDFS](#7-download-and-move-the-data-to-hdfs)
+7. [Setup python plugins](#8-setup-python-plugins)
+8. [Launch the notebook](#9-launch-the-notebook)
+9. [Run the notebook](#10-run-the-notebook)
 
 ### 1. Setup HDP platform
 
-Due to the various deployment methods of HDP we will cover the method we used when developing the code pattern, which was to use a Dockerized HDP Sandbox. This can be accomplished by performing the following steps:
+This code pattern was tested ony HDP v2.6.4, follow [this link](https://docs.hortonworks.com/HDPDocuments/HDP2/HDP-2.6.4/bk_command-line-installation/content/ch_getting_ready_chapter.html) to install and setup HDP cluster. 
 
-1. Navigate to [https://hortonworks.com/downloads/#sandbox](https://hortonworks.com/downloads/#sandbox) and choose the `Docker Linux/Mac` option. Note, you may be required to create a free account. This will download a `start-sandbox-hdp-standalone_2-6-4.sh.zip` file to your OS.
-
-    ![](images/hdp-dockerized.png)
-
-1. At this point start the Docker daemon and simply unzip the file to execute the script using:
-
-    ```
-    unzip start-sandbox-hdp-standalone_2-6-4.sh.zip
-    sh start-sandbox-hdp-standalone_2-6-4.sh
-    ```
-
-A Dockerized version of HDP v2.6.4 should now be up and running when the script is complete. The full documentation can be seen in the [HDP install guide](https://hortonworks.com/tutorial/sandbox-deployment-and-install-guide/section/3/).
+After you finish installing HDP v2.6.4, you need to disable livy server's CSRF protection: go to your Ambari server UI, click the `Spark2` on the left side panel, click the 'Configs' on the top, click on the `Advanced livy2-conf`
+change the variable `livy.server.csrf_protection.enable` from true to false, then restart the server.
 
+  ![](images/dsx-local-create-project-spark2-livy.png)
+ 
+Please check these requirements before proceeding further:
```

Comment:
Since the code pattern makes use of several python plugins (listed in <link to python plugin section>) and spark extension through solr spark connector, we strongly recommend that the components are at right version for the pattern to run smoothly.